configs/original-dst-cluster/proxy_config.yaml

```diff
@@ -7,24 +7,28 @@ static_resources:
     filter_chains:
     - filters:
       - name: envoy.http_connection_manager
-        config:
-          deprecated_v1: true
-          value:
-            stat_prefix: ingress_http
-            route_config:
-              virtual_hosts:
-              - routes:
-                - prefix: "/"
-                  timeout_ms: 0
+        typed_config:
+          "@type": type.googleapis.com/envoy.config.filter.network.http_connection_manager.v2.HttpConnectionManager
+          stat_prefix: ingress_http
+          route_config:
+            name: local_service
+            virtual_hosts:
+            - name: backend
+              domains:
+              - "*"
+              routes:
+              - match:
+                  prefix: "/"
+                route:
                   cluster: cluster1
-                domains:
-                - "*"
-                name: local_service
-            filters:
-            - name: router
-              config: {}
-            codec_type: auto
-    use_original_dst: true
+          http_filters:
+          - name: envoy.router
+            typed_config: {}
+          codec_type: auto
+    listener_filters:
+    - name: envoy.listener.original_dst
+      typed_config: {}
+    name: original_dst
```

Comment:
Is this the listener name, put it at top of the listener fields? While it is technically valid though I find it hard to read.

Reply:
I removed the name field as it isn't mandatory. I added it during testing as this file needed a large refactor to get to v2 config.